src/zenml/integrations/aws/io/__init__.py

```diff
@@ -16,4 +16,4 @@
 
 from zenml.integrations.aws.io.s3_plugin import ZenS3
 
-DEFAULT_FILESYSTEM_REGISTRY.register(ZenS3, 15)
+DEFAULT_FILESYSTEM_REGISTRY.register(ZenS3, 5)
```

Comment:
Whats this magic number? Why did we choose 5? Passing it as a keyword argument, might make it a bit more obvious when reading this.

Reply:
It's a priority, I'll change it to a kwarg. 5 was simply chosen because of @stefannica's PR which registers all of the filesystems with that priority, maybe he has some better reasoning for it other than "higher prio than the tfx/tf filesystems"